### PR TITLE
List all images at top

### PR DESCRIPTION
**Pull request by eloiferrer**
# MaRDI Pull Request

**Changes**:
- This lists all the docker images at the top of the file
- Also added a tag to deploy to mardi02

**Instructions for PR review**:
- [ ] Conceptual Review (Logic etc...) 
- [ ] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 

**Checklist for this PR**: 
- [ ] [Reviewers and Assignee specified.](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
- [ ] [All related issues are linked.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,16 +91,17 @@ x-common-variables: &wikibase_variables
...
   MW_ELASTIC_PORT: ${MW_ELASTIC_PORT:-9200}
 
 services:
   statsd:
-    image: ghcr.io/statsd/statsd
+    image: *statsd_image
 
   redis-jobrunner:
-    image: ghcr.io/mardi4nfdi/docker-redis-jobrunner
+    image: *redis_jobrunner_image
```
Comment (physikerwelt) [inline docker-compose.yml:104]: is it possible to use _ / - in the same way in the service name and image name.

File: docker-compose.yml [hunk 2]
```diff
@@ -57,11 +128,11 @@ services:
 
   redis:
     container_name: redis
-    image: redis:7
+    image: *redis_image
 
   cassandra-oai:
     hostname: cassandra-oai
-    image: cassandra:4.1
+    image: *cassandra_image
```
Comment (physikerwelt) [inline docker-compose.yml:135]: maybe call the image cassandra_oai_image?

File: docker-compose.yml [hunk 3]
```diff
@@ -115,7 +186,7 @@ services:
 
   elasticsearch-oai-setup:
     hostname: elasticsearch-oai-setup
-    image: centos
+    image: *centos_image
```
Comment (physikerwelt) [inline docker-compose.yml:189]: maybe use the service name here as well

File: docker-compose.yml [hunk 4]
```diff
@@ -266,7 +334,7 @@ services:
           - mongodb
 
   importer:
```
Comment (physikerwelt) [inline docker-compose.yml:336]: maybe rename the service to mardi_importer

File: docker-compose.yml [hunk 5]
```diff
@@ -293,7 +361,7 @@ services:
     entrypoint: "/app/start.sh"
 
   importer-api:
```
Comment (physikerwelt) [inline docker-compose.yml:363]: same

File: docker-compose.yml [hunk 6]
```diff
@@ -316,7 +384,7 @@ services:
       - traefik.http.routers.importer-api.tls.certResolver=le
 
   backup:
```
Comment (physikerwelt) [inline docker-compose.yml:386]: same

File: docker-compose.yml [hunk 7]
```diff
@@ -348,7 +416,7 @@ services:
 
   reverse-proxy:
```
Comment (physikerwelt) [inline docker-compose.yml:417]: maybe rename service to traefik

File: docker-compose.yml [hunk 8]
```diff
@@ -522,11 +589,11 @@ services:
 
   latexml:
     container_name: latexml
-    image: physikerwelt/latexml
+    image: *latexml_image
     restart: always
 
   setup_prometheus:
-    image: "ghcr.io/mardi4nfdi/docker-alpine-ext:main"
+    image: *mardi_docker_alpine
```
Comment (physikerwelt) [inline docker-compose.yml:596]: rename service or image?

**Comment by eloiferrer** [comment]
I've split docker-compose.yml in two files, including now docker-compose-extra.yml. 
docker-compose.yml contains now only the essential containers. There is not a clear line, but I've put all the APIs, analytics and so on in docker-compose-extra.yml. It should also be possible now to just run docker-compose.yml for development, to avoid running most of the extra stuff.

**Pull request merged**